web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create Route with Upstream', () => {
+  const name = {
+    upstream: 'test_upstream',
+    route: 'test_route',
+  };
+  const domSelectors = {
+    notification: '.ant-notification-notice-message',
+  };
+
+  beforeEach(() => {
+    // init login
+    cy.login();
+  });
+
+  it('should create route with upstream', () => {
+    // create a test upstream
+    cy.visit('/');
+    cy.contains('Upstream').click();
+    cy.wait(1000);
+    cy.contains('Create').click();
+
+    cy.get('#name').type(name.upstream);
+    cy.get('#desc').type('desc_by_autotest');
+    cy.get('#nodes_0_host').type('10.89.90.237');
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+
+    // go to route creat page
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.wait(1000);
+    cy.contains('Create').click();
+
+    // input name and description
+    cy.get('#name').type(name.route);
```

Comment:
`name` should be a string normally but used as an object here, chicky.

Reply:
Yes it is correct. 
![image](https://user-images.githubusercontent.com/72343596/104817116-40a83b00-585a-11eb-87d9-abf1902dd54e.png)

Reply:
no, I mean `name.route` is used like `name` is an object, actually, it's usually used as string.